programs/svm-spoke/src/instructions/token_bridge.rs

```diff
@@ -60,7 +60,7 @@ pub struct BridgeTokensToHubPool<'info> {
     #[account(mut)]
     pub local_token: UncheckedAccount<'info>,
 
-    /// CHECK: EventAuthority is checked in CCTP. Seeds must be \["__event_authority"\] (CCTP Token Messenger Minter
+    /// CHECK: EventAuthority is checked in CCTP. Seeds must be \["_event_authority"\] (CCTP Token Messenger Minter
```

Comment:
why change this? Event CPI uses `__event_authority` in its seeds

Reply:
unrelated change. have reverted.